environments/backfillEvents.toml

```diff
@@ -157,6 +488,10 @@
         deployed = 8928152
+    [contract.MCD_VAT]
```

Comment:
Are we adding the CDP_MANAGER and MCD_VAT here to avoid forgetting to add them in the future?

Reply:
Yep, just noticed that they were missing, so I though I'd add them to avoid confusion. 

Also, I think this may be leftover from a state sometime in the past where we had to make sure to configure all contract addresses, otherwise loading the config would fail. I feel like this isn't the case anymore, but wanted to make sure not to fail for that reason.